summarization/text_media_matching/text_media_matching_helper.py

```diff
@@ -125,6 +124,7 @@ def _get_distance_score_based_on_metric_type(
         )
 
         if self.distance_metric_type == "absolute-difference":
```

Comment:
nit: Could have constant here as well. Or better yet, convert to enum. But this could be handled later as well.

Reply:
Raised an issue for the same: 
https://github.com/googleinterns/stampify/issues/127